src/api/routes/faucets.ts

```diff
@@ -87,15 +96,21 @@ export function createFaucetRouter(db: DataStore): RouterWithAsync {
 
       const privateKey = process.env.FAUCET_PRIVATE_KEY || testnetKeys[0].secretKey;
 
-      // Guard condition: requests are limited to 5 times per 5 minutes.
+      const isStackingReq =
+        req.query['stacking'] === 'true' || [true, 'true'].includes(req.body['stacking']);
+
+      // Guard condition: requests are limited to x times per y minutes.
       // Only based on address for now, but we're keeping the IP in case
       // we want to escalate and implement a per IP policy
       const now = Date.now();
-      const window = 60 * 60 * 1000; // 60 minutes
+      const window = isStackingReq ? FAUCET_STACKING_WINDOW : FAUCET_DEFAULT_WINDOW;
+      const triggerCount = isStackingReq
+        ? FAUCET_STACKING_TRIGGER_COUNT
+        : FAUCET_DEFAULT_TRIGGER_COUNT;
```

Comment:
Instead of 2 consecutive `isStackingReq ?`, you could do it once and return a tuple

Reply:
Much better now! Good suggestion